.changeset/fluffy-spies-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'playroom': minor
+---
+
+Add support for vite
```

Comment:
A bit more detail on the breaking change (`bundler` is now required) as well as some info on getting `vite` working would be great here.

Reply:
agreed good catch, I love a good detailed changeset so will add it